src/core/imap/MCIMAPSession.h

```diff
@@ -123,6 +123,9 @@ namespace mailcore {
                                                    Encoding encoding, IMAPProgressCallback * progressCallback, ErrorCode * pError);
 
         virtual void fetchMessageAttachmentToFileByUID(String * folder, uint32_t uid, String * partID,
+                                                       Encoding encoding, String * outputFile,
+                                                       IMAPProgressCallback * progressCallback, ErrorCode * pError);
+        virtual void fetchMessageAttachmentToFileByUID(String * folder, uint32_t uid, String * partID,
```

Comment:
Last change: can you rename `fetchMessageAttachmentToFileByUID` to `fetchMessageAttachmentToFileByChunksByUID`.